go.mod

```diff
@@ -3,10 +3,10 @@ module github.com/cloudposse/terraform-provider-utils
 go 1.16
```

Comment:
Should this be bumped to go 1.17 ?

Reply:
did not want to do it since TF providers had some issues with Go 1.17